train_configs/debug_model.toml

```diff
@@ -38,13 +38,19 @@ gradient_accumulation_steps = 1
 seq_len = 2048
 warmup_steps = 2  # lr scheduler warm up, normally 20% of the train steps
 max_norm = 1.0  # grad norm clipping
-steps = 30
+steps = 20
 data_parallel_degree = -1
 tensor_parallel_degree = 1
 compile = false
 dataset = "chemlactica_train_mini"  # supported datasets: c4_test (2K), c4 (177M), chemlactica_train_mini (4K)
 data_process_style="chemlactica_style"
 
+[validation]
+batch_size = 1
+eval_freq = 3
+enable_val = true
+dataset = "c4_test"  # supported datasets: c4_test (2K), c4 (177M)
```

Comment:
looks like dataset works? If we can use both datasets and dataset paths, let's make sure only one of the two is passed in validate_config.

Reply:
> looks like dataset works? If we can use both datasets and dataset paths, let's make sure only one of the two is passed in validate_config.

the dataset_path is found in the build_hf_... so we need to pass both and it automatically finds the path, if You were talking about that

Reply:
That's right, I didn't realize this was the original intended functionality. Let's keep it the way it is.